.nvmrc

```diff
@@ -1 +1 @@
-14.17.4
+16.17.1
```

Comment:
Looking at the current [releases][1] page, it seems that v16 will be end-of-life in September.

Should we consider trying out v18, which is the current "Active LTS" release? It's end-of-life for support is marked as April of 2025.

[1]: https://nodejs.dev/en/about/releases/

Reply:
I tried but we have deps that do not support it. Those are gone or updated in the related PR - https://github.com/dobromir-hristov/swift-docc-render/pull/15

So I will update it there.